.github/maintainers_guide.md

```diff
@@ -1,199 +1,162 @@
 contained in the `docs` directory. Reading the Jekyll configuration in `docs/_config.yml` is helpful to understand how
-the documentation is organized and built. To build the docs, you will have to run the command `bundle exec jekyll serve` 
-in the docs directory. A prerequisite to building the docs locally is to have ruby ~2.5.3 and install the dependencies by running `bundle install`.
+the documentation is organized and built. 
 
-To build reference documentation, in the root of this repo, run `npm run ref-docs`. This will generate reference docs and put them in the `docs/_reference` directory.
-Currently, reference docs need to be built manually running this command and checked into GitHub.
+To build the docs locally, navigate to the `docs` directory. First ensure you have Ruby ~2.5.3 and install the dependencies by running `bundle install`. Then, run the command `bundle exec jekyll serve`. You will then be provided with a local URL that you can use to view the build.
+
+To build reference documentation, in the root of this repo, run `npm run ref-docs`. This will generate reference docs and put them in the `docs/_reference` directory. Currently, reference docs need to be built manually running this command and checked into GitHub.
 
 **TODO**: Update this doc once building of reference docs is automated to happen on each commit to the repo
 
 Reference docs are built by using various open source tools and formats. This includes [API-Extractor](https://api-extractor.com/), [mdast](https://github.com/syntax-tree/mdast), and [Remark](https://github.com/remarkjs/remark). [Read more about the reference docs pipeline](https://github.com/slackapi/node-slack-sdk/pull/831#issue-299509206).
 
-### Releasing
+### 🚀 Releases
+_For beta releases, see [**Beta Releases**](https://github.com/slackapi/node-slack-sdk/blob/main/.github/maintainers_guide.md#beta-releases) section below_
 
-Before releasing, it's important to understand that you may release several packages simultaneously. While `lerna` can
-offer its view on which packages have changed since the last release, you should **verify version changes yourself**
-before continuing with the release. If you find a file or set of files that should be allowed to change without
-publishing a package, it might be a good idea to add it to the `ignoreChanges` setting in `lerna.json`.
+Releasing can feel intimidaitng at first, but rest assured: if you make a mistake, don't fret! npm allows you to unpublish a release within the first 72 hours of publishing (you just won’t be able to use the same version number again). Venture on!
 
-If you make a mistake, don't fret. NPM allows you to unpublish a release within the first 72 hours of publishing, but you won’t be able to use the same version number again. Venture on!
+1. Navigate to the specific package you're releasing in the `packages/` directory.
```

Comment:
Before starting the release processes, we can have the step to check the latest status of GitHub milestones that we are going to ship as unintentionally some still may have open issues. 

If everything is ready for release, we can start the process. Otherwise, we can discuss whether we should move the remaining tasks to future releases or hold off the release until they're resolved.

Reply:
Ah, great point! I'll add it in.